testsuite/cluster-test/src/experiments/client_compatibility_test.rs

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+#![forbid(unsafe_code)]
+
+use crate::{
+    cluster::Cluster,
+    experiments::{Context, Experiment, ExperimentParam},
+};
+use async_trait::async_trait;
+use libra_logger::prelude::*;
+use std::{collections::HashSet, fmt, time::Duration};
+use structopt::StructOpt;
+use tokio::time;
+
+#[derive(StructOpt, Debug)]
+pub struct ClientCompatiblityTestParams {}
+
+pub struct ClientCompatibilityTest {}
+
+impl ExperimentParam for ClientCompatiblityTestParams {
+    type E = ClientCompatibilityTest;
+    fn build(self, _cluster: &Cluster) -> Self::E {
+        Self::E {}
+    }
+}
+
+#[async_trait]
+impl Experiment for ClientCompatibilityTest {
```

Comment:
Can you make some comment here that test itself does not do anything right now (some kind of TODO)